app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/SessionPropertiesFormatter.kt

```diff
@@ -0,0 +1,45 @@
+package nerd.tuxmobil.fahrplan.congress.utils
+
+import nerd.tuxmobil.fahrplan.congress.models.Session
+
+class SessionPropertiesFormatter {
+
+    fun getFormattedSpeakers(session: Session) =
+        with(session.speakers) {
+            when {
+                isEmpty() -> ""
+                else -> joinToString(", ")
+            }
+        }
+
+    fun getFormattedTrackLanguageText(session: Session) =
+        with(StringBuilder()) {
+            append(session.track)
+            if (session.track.isNotEmpty() && session.language.isNotEmpty()) {
+                append(" ")
+            }
+            if (session.language.isNotEmpty()) {
+                append("[")
+                append(getLanguageText(session))
+                append("]")
+            }
+            toString()
```

Comment:
This could be:

```kotlin
buildString {
    append(session.track)

    if (session.language.isNotEmpty()) {
        if (isNotEmpty()) {
            append(" ")
        }

        append("[")
        append(getLanguageText(session))
        append("]")
    }
}
```

Reply:
:heavy_check_mark: Thank you. Changed.